framework/src/transfers/MultiAppPostprocessorInterpolationTransfer.C

```diff
@@ -75,6 +75,10 @@ MultiAppPostprocessorInterpolationTransfer::MultiAppPostprocessorInterpolationTr
 void
 MultiAppPostprocessorInterpolationTransfer::execute()
 {
+  // MultiAppPostprocessorInterpolationTransfer will crash upon creating an initialSetup() for it.
+  // As such, we do the execute_on check in execute() instead.
+  checkPostProcessorExecuteOn(_postprocessor);
```

Comment:
this crash likely means your creation of an initialSetup is removing the parent class initial setup.

Can you try calling the parent class initial setup from the new initial setup ?

Reply:
that seems to have fixed it, will include in next push